app/src/main/res/layout/fragment_attendee.xml

```diff
@@ -211,8 +211,8 @@
                 <com.google.android.material.textfield.TextInputLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_marginBottom="@dimen/padding_medium"
-                    android:layout_marginTop="@dimen/padding_medium">
+                    android:layout_marginTop="@dimen/padding_medium"
+                    android:layout_marginBottom="@dimen/padding_medium">
```

Comment:
I dont think this change is necessary

Reply:
I guess I accidentally ran the code-formatted in Android Studio. I'll remove these changes right away.